samples/unit-tests/series-pie/drilldown/demo.js

```diff
@@ -1,4 +1,4 @@
-QUnit.test('Slice color after drilldown and select.', function (assert) {
+QUnit.test('Slice color after drilldown and select (#4359)', function (assert) {
```

Comment:
This test fails because in karma, there is no element with id `container_2`. There are two options for refactoring the test:
* Either first create a 2d chart, run all the tests on that then create a 3D chart in the same container and run all the tests there.
* Or, add two new div elements inside the `container` (using `document.createElement`) and add the charts to them.